packages/components/src/tooltip/test/index.tsx

```diff
@@ -15,321 +15,408 @@ import { shortcutAriaLabel } from '@wordpress/keycodes';
 import Button from '../../button';
 import Modal from '../../modal';
 import Tooltip, { TOOLTIP_DELAY } from '..';
-import cleanupTooltip from './utils/';
 
 const props = {
-	children: <Button>Button</Button>,
+	children: <Button>Tooltip anchor</Button>,
 	text: 'tooltip text',
 };
 
-describe( 'Tooltip', () => {
-	it( 'should not render the tooltip if multiple children are passed', async () => {
-		render(
-			// expected TS error since Tooltip cannot have more than one child element
-			// @ts-expect-error
-			<Tooltip { ...props }>
-				<Button>This is a button</Button>
-				<Button>This is another button</Button>
-			</Tooltip>
-		);
-
-		expect(
-			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
-		).not.toBeInTheDocument();
-	} );
-
-	it( 'should not render the tooltip if there is no focus', () => {
-		render( <Tooltip { ...props } /> );
+const expectTooltipToBeVisible = () =>
+	expect(
+		screen.getByRole( 'tooltip', { name: 'tooltip text' } )
+	).toBeVisible();
 
-		expect(
-			screen.getByRole( 'button', { name: /Button/i } )
-		).toBeVisible();
+const expectTooltipToBeHidden = () =>
+	expect(
+		screen.queryByRole( 'tooltip', { name: 'tooltip text' } )
+	).not.toBeInTheDocument();
 
-		expect(
-			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
-		).not.toBeInTheDocument();
-	} );
+const waitForTooltipToShow = ( timeout = TOOLTIP_DELAY ) =>
+	waitFor( () => expectTooltipToBeVisible(), { timeout } );
 
-	it( 'should render the tooltip when focusing on the tooltip anchor via tab', async () => {
-		const user = userEvent.setup();
+const waitForTooltipToHide = () => waitFor( () => expectTooltipToBeHidden );
 
-		render( <Tooltip { ...props } /> );
+const hoverOutside = async () => {
+	await hover( document.body );
+	await hover( document.body, { clientX: 10, clientY: 10 } );
+};
 
-		await user.tab();
+describe( 'Tooltip', () => {
+	describe( 'basic behavior', () => {
+		it( 'should not render the tooltip if multiple children are passed', async () => {
+			render(
+				// expected TS error since Tooltip cannot have more than one child element
+				// @ts-expect-error
```

Comment:
Small tip: place the comment after the TS directive. If it fails (meaning, the error isn't thrown anymore), TSC will display the entire comment rather than just the directive.

```suggestion
				// @ts-expect-error expected TS error since Tooltip cannot have more than one child element
```